src/lib/components/SideNav/SideNav.scss

```diff
@@ -0,0 +1,13 @@
+.usa-sidenav__item.ldaf-sidenav-item .usa-link {
+  display: flex;
+  align-items: center;
+  .ldaf-sidenav-link-text {
+    width: 75%;
+  }
+  .ldaf-sidenav-icon-container {
+    width: 25%;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+  }
+}
```

Comment:
Styling here is just focused on properly rendering the collapse / expand icons next to links that have child pages.